modules/ROOT/pages/provisioning-vultr.adoc

```diff
@@ -0,0 +1,37 @@
+= Provisioning Fedora CoreOS on Vultr
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on Vultr.  Fedora CoreOS images are currently not published directly on Vultr, so you must download a Fedora CoreOS Vultr image and upload it to your Vultr account as a https://www.vultr.com/docs/requirements-for-uploading-an-os-iso-to-vultr[custom image].
+
+== Prerequisites
+
+Before provisioning a FCOS machine, you must have an Ignition configuration file that sets SSH authorized keys for the core user. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File]. While the Vultr documentation mentions cloud-init and scripts, FCOS does not support cloud-init or the ability to run scripts from user-data. It accepts only Ignition configuration files.
+
+
+You also need to have access to a Vultr account.  The examples below use the https://github.com/vultr/vultr-cli[vultr-cli] command-line tool.
+
+== Creating a Vultr custom snapshot
+
+Fedora CoreOS is designed to be updated automatically, with different schedules per stream.
+
+. Once you have picked the relevant stream, find the corresponding Vultr image on the https://getfedora.org/coreos/download?tab=cloud_operators[download page] and copy the URL of the Download link.
+
+. Create the custom image:
++
+.Example uploading FCOS to a Vultr custom snapshot
+[source, bash]
+----
+export VULTR_API_KEY=<token>
+vultr-cli snapshot create-url -u <download-url>
```

Comment:
is there any option here to do verification of the image (maybe via providing a checksum)?

Reply:
That would be sweet, but the CLI just takes any URL to download, same as their UI. No checksum verification.

```
Usage:
  vultr-cli snapshot create-url [flags]

Flags:
  -h, --help         help for create-url
  -u, --url string   Remote URL from where the snapshot will be downloaded.
```